src/musicbrainz/web/coverartarchiveimagetask.cpp

```diff
@@ -0,0 +1,92 @@
+#include "musicbrainz/web/coverartarchiveimagetask.h"
+
+#include <QMetaMethod>
+
+#include "defs_urls.h"
+#include "network/httpstatuscode.h"
+#include "util/assert.h"
+#include "util/logger.h"
+#include "util/thread_affinity.h"
+#include "util/versionstore.h"
+
+namespace mixxx {
+
+namespace {
+
+const Logger kLogger("CoverArtArchiveImageTask");
+
+QNetworkRequest createNetworkRequest(const QUrl& thumbnailUrl) {
+    DEBUG_ASSERT(thumbnailUrl.isValid());
+    QNetworkRequest networkRequest(thumbnailUrl);
+    return networkRequest;
+}
+
+} // anonymous namespace
+
+CoverArtArchiveImageTask::CoverArtArchiveImageTask(
+        QNetworkAccessManager* networkAccessManager,
+        const QUrl& coverUrl,
+        QObject* parent)
+        : network::WebTask(
+                  networkAccessManager,
+                  parent),
+          m_ThumbnailUrl(coverUrl) {
```

Comment:
```suggestion
          m_coverUrl(coverUrl) {
```

This makes me think if the class is actually just a 'FetchImage' class